apis/duck/v1alpha1/conditions_types.go

```diff
@@ -187,7 +187,7 @@ func (t *KResource) Populate() {
 }
 
 // GetListType implements apis.Listable
-func (r *KResource) GetListType() runtime.Object {
+func (t *KResource) GetListType() runtime.Object {
```

Comment:
Why does this need `t`?

Reply:
There is a lint test that errors when a receiver has a different name within the same file. The receiver is named `t` on line 177: https://github.com/knative/pkg/pull/351/files/f35a510212e1636b012c3bee893f6043c18bde95#diff-04abbb75ad1eb1e871f98943273acd40L177

Reply:
I think the question is why we need it at all?

Reply:
Ah thanks for the clarification. My understanding is that this implements the Listable interface: https://github.com/trshafer/pkg/blob/f35a510212e1636b012c3bee893f6043c18bde95/apis/interfaces.go#L48. So the objects can be used here: https://github.com/trshafer/pkg/blob/f35a510212e1636b012c3bee893f6043c18bde95/apis/duck/typed.go#L49

Reply:
To further clarify, why not this?
```suggestion
func (*KResource) GetListType() runtime.Object {
```

Reply:
Great suggestion. Thanks.